crates/bevy_ecs/src/observer/mod.rs

```diff
@@ -84,6 +84,17 @@ impl<'w, E, B: Bundle> Trigger<'w, E, B> {
     }
 }
 
+impl<'w, E: Debug, B: Bundle + Debug> Debug for Trigger<'w, E, B> {
```

Comment:
```suggestion
impl<'w, E: Debug, B: Bundle> Debug for Trigger<'w, E, B> {
```

Which also means we can remove the `Debug` bound on the targeted `Bundle` type.

Reply:
Good catch! Even if we keep the `PhantomData`, I don't think this bound is necessary

Reply:
Done!